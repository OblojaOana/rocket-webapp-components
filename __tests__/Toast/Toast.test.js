import React from "react";
import renderer from "react-test-renderer";
import ToastContainer from "../../components/Toast/ToastContainer";

describe("Render ToastContainer", () => {
  it("render correctly ToastContainer component", () => {
    const component = renderer.create(<ToastContainer />);

    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
