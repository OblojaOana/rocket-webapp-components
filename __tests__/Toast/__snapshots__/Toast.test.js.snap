// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render ToastContainer render correctly ToastContainer component 1`] = `
<div
  className="Toastify"
/>
`;
